main.go

```diff
@@ -35,17 +37,20 @@ func (mf *mapFlags) Set(value string) error {
 
 func main() {
 	renameStructs := mapFlags{kv: make(map[string]string)}
+	renameFields := mapFlags{kv: make(map[string]string)}
 	var useMessagePack bool
 
 	flag.Var(&renameStructs, "rename", "Override generated name of structure; use '-rename old=new'.")
+	flag.Var(&renameFields, "renameFields", "Override generated name of structure; use '-renameFields old=new'.")
 	flag.BoolVar(&useMessagePack, "msgp", false, "generate MessagePack serialization methods")
 
 	flag.Parse()
 
 	args := flag.Args()
 
 	if len(args) < 1 {
-		fmt.Printf("Usage: %s [-msgp] <JSON Schema Path> [output directory]\n", path.Base(os.Args[0]))
+		fmt.Printf("Usage: %s [-msgp] [-rename[ [-renameFields] <JSON Schema Path> [output directory]\n", path.Base(os.Args[0]))
```

Comment:
`[-rename[` should be `[-rename]` maybe even `[-rename k=v]` to indicate that a value must follow it. Also FWIW the pflag package here supports flag.StringArrayVar which may be more appropriate here to allow multiple options.

Reply:
Thanks, I can fix that

Reply:
adding pflag (https://github.com/spf13/pflag) can be done in another PR, later.